### DISCUSSION TITLE
`approximate` vs `approximative`

### DISCUSSION THREAD

**Discussion by tukusejssirs** [romcal/romcal#308] [Ideas]
@emagnier, I have just found out that you added `dateOfDeathIsApproximative` property used in `lib/catalog/martyrology.ts`. Nice work! :wink:

However, I haven’t heard yet of `approximative` word in English (altought it [does exist](https://www.lexico.com/en/definition/approximative)). I would use [`approximate`](https://www.lexico.com/en/definition/approximate), which _might_ be better (I am not sure).

What made you choose that word? French? :wink:

**Comment by emagnier** [comment]
Yes, some words both exists in French and English, but do not always have exactly the same meaning, usage, or spelling. And it sometime happens to me to not pay attention to them when I'm focussing on something else 😄 

For sure, now that you flag it, I also think that `approximate` is better suited here.

FYI, all the actual properties are described here:

https://github.com/romcal/romcal/blob/955dd6c01ffd68f47717002c606539ed81f7f056/lib/types/martyrology.ts#L3-L61

But this is just an initial version. Now, we could probably take another look to #208 and refine the metadata of this martyrology catalog, and the way how data are defined (e.g. the `dateOfDeath` definitions).

**Reply by tukusejssirs** [reply]
Taking a closer look at #208 is a great idea. When I was working on #309, the following came to my mind: add some metadata based on other properties. For example, based on the `DoctorOfTheChurch` title, we could add the personal title of that person (e.g. `Doctor of Unity` for St Irenaeus). Similarly, cardinals have their titular sees in Rome. Martyrs usually have there persecutor name listed in Martyrologium Romanum, therefore I suggest to list it. And not to forget about the data required by the Ordos.

Anyway, you were thinking about moving #208 to GitHub Discussions. Have you made your mind yet? Where could we discuss this? :wink:

**Reply by emagnier** [reply]
Thanks for the reminder. #208 has been moved to a discussion: #310.
